azurerm/internal/services/containers/kubernetes_addons.go

```diff
@@ -84,6 +87,39 @@ func schemaKubernetesAddOnProfiles() *schema.Schema {
 					},
 				},
 
+				"ingress_appgw": {
```

Comment:
could we not abbreviate this?
```suggestion
				"ingress_app_gateway": {
```